Code/C++/readme.md

```diff
@@ -1,3 +1,4 @@
  # Codes in the C++ language
 
-Unique Permutations of String ----> [C++](/Code/C++/unique_permutaions_of_string.cpp)
\ No newline at end of file
+Unique Permutations of String ----> [C++](/Code/C++/unique_permutaions_of_string.cpp)
+Geek-onacci Number----> [C++](/Code/C++/Geek-onacciNumber.cpp)
```

Comment:
make these changes in Algo-Tree/Algorithm/Number_theory/ and not in Code/C++/readme.md

Reply:
@Rohan27s  ReadMe updated please check